### ISSUE TITLE
Save as a project file

### ISSUE THREAD

**Issue by achavazza** [oklai/koala#356]
save the folder, paths and configs, as a file.

this file could work as a setup for a project, could be useful to **share** it or for **save and to use later**

**Comment by hahnzhu** [comment]
can `koala-config.json` fill your request？ [Using-project-settings](https://github.com/oklai/koala/wiki/Using-project-settings)

**Comment by fri3ndly** [comment]
I agree with achavazza. koala-config.json sounds like it could work, but is there any way that this file could be auto generated by the app based  on your settings? My team are all working from the same shared network folder and it wouldbe so handy if the person initially setting up the Koala project auto generated the file which anyone else connecting to would import and then amend too if need be automatically.

**Comment by fri3ndly** [comment]
Further to the above I have been looking into this and actually  the koala-config.json method does work very well. I have it set up exactly as required so only the one less file is being compiled and when others load up the project they see exactly what I see - great!

One thing I am unsure of how to do is set the settings for compiling javascript files - any ideas? They are not being auto compiled at the moment
